src/js/components/TaskDetail.js

```diff
@@ -166,12 +166,17 @@ class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
       </ul>
     );
 
+    const mediaWrapperClassNames = {
+      'media-object-spacing-narrow': false,
+      'media-object-offset': false
+    };
+
     return (
       <PageHeader
         icon={taskIcon}
         subTitle={TaskStates[task.state].displayName}
         navigationTabs={tabs}
-        mediaWrapperClassName="media-object-spacing-wrapper"
+        mediaWrapperClassName={mediaWrapperClassNames}
```

Comment:
NB this can safely be merged, but won't actually work until a bug in `PageHeader` is fixed (#188)